### ISSUE TITLE
Connecting to Non Ultimaker Printers

### ISSUE THREAD

**Issue by meldrex5** [Ultimaker/Cura#2135]
If Cura is the first program opened after connecting a printer it will automatically connect to it.  This alone is annoying but then there is no option to disconnect from it, you have to close Cura, connect with the other interface then reopen Cura.

**Comment by Ghostkeeper** [comment]
Also happens with the Ultimaker Original.

**Comment by astecme** [comment]
Indeed. Cura does not return to the main program after finishing and the only way out is to shut Cura down.

**Comment by Ghostkeeper** [comment]
Oh, well you should be able to switch back to the settings view while it's printing... Aren't you?

**Comment by Ghostkeeper** [comment]
Our project manager just removed this from our planning because it's older than 12 weeks.

**Comment by rlesac** [comment]
It would be lovely to be able to disconnect from the printer without shutting down Cura. 
Please reconsider.

**Comment by metaxis** [comment]
an option to manually dis/connect vs autoconnect would be great.

**Comment by yodor** [comment]
I keep pushing the reset button of my printer to help Cura recognize it. Would be nice to have port/speed/connect button panel.

**Comment by GregValiant** [comment]
5.12 has a "Preference" to disable USB Printing by default.  There are a lot of peripherals now (earbuds and whatnot) that are ocnigured as serial devices and they caused a lot of confusion.  I'll go ahead and close this.
